### PR TITLE
Add janitor scenario for custom projects.

### LINKED ISSUE #8151: make janitor scenario support clean customized projects
aka --mode=custom, --project=proj1,proj2,proj3...

(in case I forgot)
/assign

### PR DESCRIPTION
**Pull request by amwat**
Fixes #8151.

### CODE REVIEW

File: scenarios/kubernetes_janitor.py [hunk 1]
```diff
@@ -138,12 +138,17 @@ def check_ci_jobs():
...
     """Run janitor for each project."""
     if mode == 'pr':
         check_predefine_jobs(PR_PROJECTS, ratelimit)
     elif mode == 'scale':
         check_predefine_jobs(SCALE_PROJECT, ratelimit)
+    elif mode == 'custom':
+        projs = str.split(projects, ',')
+        # Setting expiry for all the projects to 3 hours
+        jobs = dict(zip(projs, [3]*len(projs)))
+        check_predefine_jobs(jobs, ratelimit)
```
[the remark this answers is not in the document]
Comment (amwat) [reply scenarios/kubernetes_janitor.py]: `check_predefine_jobs` basically just iterates anyway. Also making the flag a dict of project:age seems awful.

[the remark this answers is not in the document]
Comment (krzyzacy) [reply scenarios/kubernetes_janitor.py]: I mean make flag as --age=int, then for proj in projs, clean_project(proj, hours=age)?

[the remark this answers is not in the document]
Comment (amwat) [reply scenarios/kubernetes_janitor.py]: Done.

File: scenarios/kubernetes_janitor.py [hunk 2]
```diff
@@ -160,10 +164,16 @@ def main(mode, ratelimit):
...
         help='Which type of projects to clear')
     PARSER.add_argument(
         '--ratelimit', type=int,
         help='Max number of resources to clear in one gcloud delete call (passed into janitor.py)')
+    PARSER.add_argument(
+        '--projects', type=str,
+        help='Comma separated list of projects to clean up. Only applicable in custom mode.')
+    PARSER.add_argument(
+        '--age', type=int,
+        help='Expiry age for projects. Only applicable in custom mode.')
```
Comment (krzyzacy) [inline scenarios/kubernetes_janitor.py]: `Expiry age for projects, in hours`

**Pull request merged**